lld/test/wasm/unresolved-symbols.s

```diff
@@ -53,9 +58,10 @@
 # IGNORE-NEXT:        Name:            get_func_addr
 
 ## --import-undefined should handle unresolved funtions symbols
-# by importing them but still report errors/warning for missing data symbols.
-# `--allow-undefined` should behave like `--import-undefined` +
-# `--unresolve-symbols=ignore`
+## by importing them but still report errors/warning for missing data symbols.
+## `--allow-undefined` should behave like `--import-undefined` +
+## `--unresolve-symbols=ignore`
+# RUN: wasm-ld %t1.o -o %t3.wasm --import-undefined --unresolved-symbols=ignore-all
```

Comment:
was it intentional to duplicate this line (should it have been `--allow-undefined`?)

Reply:
Oops, reverted